app/components/speechBubble/speechBubble.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './speechBubble.css';
+
+interface SpeechBubbleProps {
+  text: string;
+  direction: 'left' | 'right';
+}
+
+const SpeechBubble: React.FC<SpeechBubbleProps> = ({ text, direction }) => {
+  const bubbleStyles = {
+    alignSelf: direction === 'left' ? 'flex-start' : 'flex-end',
+    backgroundColor: direction === 'left' ? '#fff' : '#F3F3F3',
+    color: direction === 'left' ? '#000' : '#333',
+  };
+
+  return (
+    <div className={`speech-bubble ${direction}`} style={bubbleStyles}>
```

Comment:
As you already defined a CSS class for the direction the upper bubbleStyles can be in, these CSS classes